src/components/Copyable.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { PropsWithChildren, useRef } from 'react'
+import { copyToClipboard } from '../utils'
+
+export interface CopyableProps {
+  value: string
+  onSuccess?: () => void
+  onError?: (e: Error) => void
+  className?: string
+}
+
+export function Copyable({
+  value,
+  onSuccess,
+  onError,
+  className,
+  children,
+  ...props
```

Comment:
I don't think `...props` is doing anything here

Reply:
Now it does - passing other properties, e.g. "disabled" : D
Thanks!